### ISSUE TITLE
Contracts Buyer: Should it be award buyer instead?

### ISSUE THREAD

**Issue by yolile** [open-contracting/ocds-extensions#131]
From standard development discussion:

Currently, for modelling framework agreements with direct call offs the documentation says that the information about the framework set up should go in the award block and the information about the purchase orders in the contract block. As different buyers can make purchase orders, and as the award block is already been used, the extension adds the buyer block to the contract block to distinguish who buys what. 

This recommendation is about to change in 1.1.5 and in 1.2 as well, discouraging the use of the same process for all the purchases orders in the first case and the use of the award block for the framework set up in the second. 

Taking into account that in all the cases normally if there are multiple suppleirs on the FA, then there is some formula/method for determining to whom to procure the goods/services. The result of that formula/method is, conceptually, the award. So the award can have the items, suppliers, value as usual. So, all the information is at the award level, so probably the buyer should be in the award too instead of in the contract.

So the suggestion is to update the documentation and the schema of this extension to use the buyer block at the award level and not at the contract level.

cc @jpmckinney

**Comment by jpmckinney** [comment]
Agreed. We can go ahead and add `buyer` (or `buyers`?) to `Award` in the extension, and update the guidance. We can keep `buyer` on `Contract` for backwards-compatibility and any niche use cases.
